Gems/MultiplayerCompression/Code/Source/LZ4Compressor.cpp

```diff
@@ -56,11 +56,16 @@ namespace MultiplayerCompression
         AZ_Warning("Multiplayer Compressor", compDataSize >= compWorstCaseSize, "Outbuffer size (%lu B) passed to Compress() is less than estimated worst case (%lu B)", compDataSize, compWorstCaseSize);
 
         // Note that this returns a non-negative int so we are narrowing into a size_t here
-        compSize = LZ4_compressHC(reinterpret_cast<const char*>(uncompData), reinterpret_cast<char*>(compData), static_cast<int>(uncompSize));
+        compSize = LZ4_compress_HC(
+            reinterpret_cast<const char*>(uncompData), 
+            reinterpret_cast<char*>(compData), 
+            static_cast<int>(uncompSize),
+            static_cast<int>(compDataSize),
```

Comment:
With this here you can remove the [[maybe_unused]] on the compDataSize.

Reply:
If there are any other changes needed I will also make this change.